bin/configs/csharp-echo-api.yaml

```diff
@@ -0,0 +1,9 @@
+generatorName: csharp
```

Comment:
Can you rename the file to start with csharp-restsharp to match the format of the other files?